polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -53,14 +62,27 @@ public String catalogConfig() {
   }
 
   T cast(Object value) {
```

Comment:
I kinda want to call this apply() now instead of cast ... :) 

Just because it does dual duty now